docs/self-managed/setup/install.md

```diff
@@ -74,7 +74,7 @@ This size is comparable with the Production-S cluster plan in [Camunda 8 SaaS](h
 Before deploying Camunda using Helm, you need the following:
 
 - Kubernetes cluster: either local, cloud platform, or on-premises.
-- [Helm](https://helm.sh/docs/intro/install/).
+- [Helm](https://helm.sh/docs/intro/install/) version 3.13+.
```

Comment:
Is this the Helm version or the Helm CLI version? Are they different?

Reply:
It's easier maintenance-wise to keep the version info on the supported environments page only. I would remove it from here, then.